.pre-commit-config.yaml

```diff
@@ -58,15 +58,15 @@ repos:
           - pep8-naming==0.13.3
           - flake8-pyproject==1.2.2
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v2.7.1"
+    rev: "v3.0.0-alpha.4"
```

Comment:
No alpha release please

Reply:
this comment belongs to #132 (upon which this is based), not here.

Reply:
... and without basing it on #132, github CI does not work, see #133.